header.mk

```diff
@@ -124,7 +124,7 @@ CPPFLAGS             = $(CPPFLAGS.arch) $(CPPFLAGS.includes) $(CPPFLAGS.debug) $
 CFLAGS.warnings      = -std=c99 -pedantic -Wall -Wno-array-bounds -Werror=implicit -Werror=uninitialized -Werror=return-type
 CFLAGS.arch          = -m32 -march=i686
 CFLAGS.optimization  = -O3
-CFLAGS.others        = -ffreestanding -fno-pie -fno-common -fno-omit-frame-pointer -fno-delete-null-pointer-checks
+CFLAGS.others        = -ffreestanding -fno-pie -fno-common -fno-omit-frame-pointer -fno-delete-null-pointer-checks -fno-asynchronous-unwind-tables
```

Comment:
Adding `-fno-asynchronous-unwind-table` so the unneeded `.eh_frame` section does not get generated. This was done specifically to reduce the size of the setup code but reducing size in this way benefits the whole kernel.